lib/Parse/ParseStmt.cpp

```diff
@@ -637,6 +643,12 @@ ParserResult<Stmt> Parser::parseStmt() {
     if (LabelInfo) diagnose(LabelInfo.Loc, diag::invalid_label_on_stmt);
     if (tryLoc.isValid()) diagnose(tryLoc, diag::try_on_stmt, Tok.getText());
     return parseStmtPoundAssert();
+  case tok::l_brace:
+    if (tryLoc.isValid()) diagnose(tryLoc, diag::try_on_stmt, Tok.getText());
+    SourceLoc colonLoc = Tok.getLoc();
```

Comment:
Sorry for nitpicking:
```suggestion
    SourceLoc braceLoc = Tok.getLoc();
```

Reply:
Fixed.